lib/src/view/home/home_tab_screen.dart

```diff
@@ -527,7 +527,7 @@ class _HelloWidget extends ConsumerWidget {
             ),
             const SizedBox(width: 5.0),
             Text(
-              'Hello${user != null ? ', ' : ''}',
+              '${context.l10n.mobileHello}${user != null ? ', ' : ''}',
```

Comment:
@veloce This should probably be fine for most languages, but I'm sure there are some languages where "Hello, %s" does not make any sense, we might want to revisit this at some point

Reply:
@tom-anders that is because we use the `UserFullNameWidget`.

Now that I think of it we should make it right from the start and have a parameterised message such as `helloX` : `Hello, <name>`.

We can adapt the HelloWidget later to show the flair for instance.

Reply:
Ok, so the string is now 'Hello %s' and I added a special case in the translation generation, making sure that the %s is **not** converted to a function parameter. This way, we can instead split the (localized) string at '%s' and put the UserNameWidget between the two resulting parts.